R/build.R

```diff
@@ -27,19 +40,18 @@ inlineCxxPlugin <- function() {
    )
 }
 
-mtuneFlags <- function() {
-   switch(Sys.info()[["sysname"]],
-          "Linux"   = "-mtune=native",
-          "Darwin"  = "-mtune=core2",
-          "Windows" = "-mtune=core2",
-          ""
-   )
-}
-
-tbbCxxFlags <- function() {
-   flags <- "$(CXX1XSTD)"
+tbbCxxFlags <- function(simd = TRUE) {
```

Comment:
I wonder if the SIMD stuff should be in its own function, e.g. `simdCxxFlags()`, and then the `cxxFlags()` argument would be the `paste()`d output from each?